pyke/tests/test_lightcurve.py

```diff
@@ -26,6 +26,23 @@ def test_KeplerLightCurve():
     assert kplc.mission == 'Kepler'
 
 
+@pytest.mark.parametrize("quality_bitmask,answer",[('hard',2661),
+                                            ('conservative',2713),
```

Comment:
space after commas. You might want to shrink things here. No need to have one option
per line.